ons_alpha/jinja2/component_overrides/footer/_macro.njk

```diff
@@ -152,46 +154,68 @@
                                                 })
-                                            }}{{ params.OGLLink.post | default(', oni bai y nodir fel arall') }}
+                                            }}{{ params.oglLink.post | default(', oni bai y nodir fel arall') }}
                                         {% else %}
-                                            {{ params.OGLLink.pre | default('All content is available under the') }}
+                                            {{ params.oglLink.pre | default('All content is available under the') }}
                                             {{
                                                 onsExternalLink({
-                                                    "url": params.OGLLink.url | default('https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/'),
-                                                    "linkText": params.OGLLink.link | default('Open Government Licence v3.0'),
+                                                    "url": params.oglLink.url | default('https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/'),
+                                                    "text": params.oglLink.text | default('Open Government Licence v3.0'),
                                                     "classes": 'footer__external-link'
                                                 })
-                                            }}{{ params.OGLLink.post | default(', except where otherwise stated') }}
+                                            }}{{ params.oglLink.post | default(', except where otherwise stated') }}
                                         {% endif %}
                                     </div>
                                 {% endif %}
                             </div>
                         {% endif %}
-                        {% if not params.poweredBy %}
-                            <a
-                                class="ons-footer__poweredBy-link"
-                                {% if lang == "cy" %}href="https://cy.ons.gov.uk/"{% else %}href="https://www.ons.gov.uk/"{% endif %}
+                        {% if params.footerLogo %}
+                            {% if not params.footerLogo.logos.logo1 %}
+                                {%
+                                    set logo1 = {
+                                        'logoUrl': 'https://cy.ons.gov.uk/' if lang == 'cy' else 'https://www.ons.gov.uk/',
+                                        'logoImage': onsLogo | safe
+                                    }
+                                %}
+                                {% set logos = [logo1, params.footerLogo.logos.logo2] %}
+                            {% else %}
+                                {% set logos = [params.footerLogo.logos.logo1, params.footerLogo.logos.logo2] %}
+                            {% endif %}
+
```

Comment:
I don't think we've nailed the logo changes just yet... it feels like something that would benefit from better up-front value prep, and less code here. Let's make a note to come back to it later.